src/store/socket.js

```diff
@@ -3,27 +3,29 @@ import { gotMessages } from '../store/messages/actions'
 
 let token = ''
 
-const socket = io('http://localhost:5000');
+const socket = io('http://localhost:5000', {
+    query: { token: token }
+});
 socket.connect();
 
 
-export const openChat = users => {
+export function openChat(users) {
```

Comment:
If you use arrow-functions here.
```
    handleSubmit = () => {
        const { name } = this.state
        const { onSearch } = this.props

        onSearch(name)
    }
```
You must use them throughout the app, not just in a single file.